minigalaxy/installer.py

```diff
@@ -167,6 +170,35 @@ def copy_thumbnail(game):
     return error_message
 
 
+def create_applications_file(game):
+    error_message = ""
+    preferences_switch = Config.get("create_applications_file")
+    if game.platform == "linux" and preferences_switch:
+        path_to_shortcut = os.path.join(APPLICATIONS_DIR, game.name+".desktop")
+        # Create desktop file definition
+        desktop_context = {
+            "game_bin_path": os.path.join(game.install_dir, 'start.sh'),
+            "game_name": game.name,
+            "game_icon_path": os.path.join(game.install_dir, 'support/icon.png')
```

Comment:
Hmm, I will need to check if this icon file exists also on Windows games.

Reply:
The support/icon.png? Good point, in Heroes of Might and Magic it exists, but it's .ico. I'll check some other games.

Also, for the time being Wine auto-creates the .desktop files and we should skip generating it for Windows games anyway.

Reply:
Ok, I did some tests and the WINE-generated shortcuts do not work. I'll think how to replace them via minigalaxy.